README.md

```diff
@@ -37,4 +39,9 @@ with an `(I)` appended e.g. `EMIS (I)`. The addition of `(I)` represents a GP
 that is now using the Informatica system.
 
 Note: The list above was created by running
-`jq -c '[.[].Supplier] | unique ' output/pomi.json`
+`jq -c '[.[].Supplier] | unique ' html/json/pomi.json`
+
+The json files created will be available via the container's webserver at
```

Comment:
ETL has been running since yesterday and still getting a '404' on records.json.

There was an error on load (need to scroll right to see it) which I thought the process had recovered from:

```
etl_1       | 2017-05-30 14:37:04,835 CRIT Supervisor running as root (no user in config file)
etl_1       | 2017-05-30 14:37:04,842 INFO supervisord started with pid 1
etl_1       | 2017-05-30 14:37:05,846 INFO spawned: 'nginx' with pid 7
etl_1       | 2017-05-30 14:37:05,849 INFO spawned: 'etl' with pid 8
etl_1       | {"name":"pomi-etl","hostname":"94c98b7958bf","pid":8,"level":30,"msg":"Scheduling job with rule '38 14 * * *'","time":"2017-05-30T14:37:06.026Z","v":0}
etl_1       | 2017-05-30 14:37:07,033 INFO success: nginx entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
etl_1       | 2017-05-30 14:37:07,034 INFO success: etl entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
etl_1       | {"name":"pomi-etl","hostname":"94c98b7958bf","pid":8,"level":30,"msg":"Starting download and transformation of BOOKING data","time":"2017-05-30T14:38:00.053Z","v":0}
etl_1       | {"name":"pomi-etl","hostname":"94c98b7958bf","pid":8,"level":30,"msg":"Starting download and transformation of SCRIPTS data","time":"2017-05-30T14:38:00.058Z","v":0}
etl_1       | {"name":"pomi-etl","hostname":"94c98b7958bf","pid":8,"level":30,"msg":"Starting download and transformation of RECORDS data","time":"2017-05-30T14:38:00.058Z","v":0}
etl_1       | {"name":"pomi-etl","hostname":"94c98b7958bf","pid":8,"level":50,"msg":"Processing failed { Error: getaddrinfo EAI_AGAIN indicators.hscic.gov.uk:443\n    at Object.exports._errnoException (util.js:1050:11)\n    at errnoException (dns.js:33:15)\n    at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:73:26)\n  code: 'EAI_AGAIN',\n  errno: 'EAI_AGAIN',\n  syscall: 'getaddrinfo',\n  hostname: 'indicators.hscic.gov.uk',\n  host: 'indicators.hscic.gov.uk',\n  port: 443 }","time":"2017-05-30T14:38:05.103Z","v":0}
etl_1       | Downloading the BOOKING file took: 50493.840ms
etl_1       | {"name":"pomi-etl","hostname":"94c98b7958bf","pid":8,"level":30,"msg":"html/json/booking.csv saved","time":"2017-05-30T14:38:50.890Z","v":0}
etl_1       | Downloading the SCRIPTS file took: 51005.848ms
etl_1       | {"name":"pomi-etl","hostname":"94c98b7958bf","pid":8,"level":30,"msg":"html/json/scripts.csv saved","time":"2017-05-30T14:38:51.377Z","v":0}
etl_1       | Removing redundant columns from BOOKING took: 22800.550ms
etl_1       | {"name":"pomi-etl","hostname":"94c98b7958bf","pid":8,"level":30,"msg":"Records processed for BOOKING data column removal: 319524","time":"2017-05-30T14:39:13.692Z","v":0}
etl_1       | Getting latest period took: 0.949ms
etl_1       | {"name":"pomi-etl","hostname":"94c98b7958bf","pid":8,"level":30,"msg":"Latest period is 28/02/2017","time":"2017-05-30T14:39:13.693Z","v":0}
etl_1       | Removing redundant columns from SCRIPTS took: 22450.307ms
etl_1       | {"name":"pomi-etl","hostname":"94c98b7958bf","pid":8,"level":30,"msg":"Records processed for SCRIPTS data column removal: 320169","time":"2017-05-30T14:39:13.828Z","v":0}
etl_1       | Getting latest period took: 0.188ms
etl_1       | {"name":"pomi-etl","hostname":"94c98b7958bf","pid":8,"level":30,"msg":"Latest period is 28/02/2017","time":"2017-05-30T14:39:13.828Z","v":0}
etl_1       | Removing old records for BOOKING took: 5115.591ms
etl_1       | {"name":"pomi-etl","hostname":"94c98b7958bf","pid":8,"level":30,"msg":"Current record count: 14880","time":"2017-05-30T14:39:18.809Z","v":0}
etl_1       | {"name":"pomi-etl","hostname":"94c98b7958bf","pid":8,"level":30,"msg":"Old record count: 305288","time":"2017-05-30T14:39:18.809Z","v":0}
etl_1       | Download and transforming BOOKING data took: 78754.571ms
etl_1       | Removing old records for SCRIPTS took: 5055.330ms
etl_1       | {"name":"pomi-etl","hostname":"94c98b7958bf","pid":8,"level":30,"msg":"Current record count: 14880","time":"2017-05-30T14:39:18.884Z","v":0}
etl_1       | {"name":"pomi-etl","hostname":"94c98b7958bf","pid":8,"level":30,"msg":"Old record count: 305288","time":"2017-05-30T14:39:18.884Z","v":0}
etl_1       | Download and transforming SCRIPTS data took: 78826.673ms
etl_1       | Converting CSV to JSON for BOOKING took: 429.117ms
etl_1       | Converting CSV to JSON for SCRIPTS took: 361.305ms
```

I've rerun this and now I no longer get the '404'. I guess the 404 is a consequence of it failing the first time it runs when it does not have the previous version to serve. Not sure what the db loader does ifone of the set of jason documents is not available.